tests/filecheck/mlir-conversion/builtin_attrs.mlir

```diff
@@ -29,6 +29,22 @@
 
   // CHECK: 42.0 : f32
 
+  "func.func"() ({}) {function_type = () -> (), value = true, sym_name = "true_attr"} : () -> ()
```

Comment:
Just running this test does not actually test the printing for the xdsl version, right? This parses xdsl, prints it as mlir and reparses it as mlir, to then reprint it as mlir, so printing with xdsl is not actually involved.

Reply:
Indeed, and I also realized that my tests were wrong...
I was getting the `true` from the function name, instead of the attribute.